src/App.tsx

```diff
@@ -17,18 +70,33 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onFilter={setFilterBy}
+                onQuery={setQuery}
+                query={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+              {!isLoading && (
+                <TodoList
```

Comment:
the ternary operator got offended at you because of this =)) 

condition ? trueExpression : falseExpression